graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/DgsRestController.kt

```diff
@@ -62,9 +62,6 @@ import org.springframework.web.multipart.MultipartFile
 @RestController
 open class DgsRestController(open val dgsQueryExecutor: DgsQueryExecutor) {
 
-    open val logger: Logger = LoggerFactory.getLogger(DgsRestController::class.java)
```

Comment:
@bono007, any chance you recall any reason to make the logger `open`?

Reply:
@berngp sorry just now getting back to this comment (been a crazy 24 hrs). I do not remember why. Its possible I had a reason and forgot by now OR I copy/pasta the logger from elsewhere.

Reply:
No worries. Hope all is well now.